src/app/ratings.service.ts

```diff
@@ -0,0 +1,40 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable, observable } from 'rxjs';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class RatingsService {
+  private REST_API_SERVER = 'http://localhost:3000/products';
+  // private REST_API_SERVER_MENU = 'http://localhost:3000/menu'; 
+
+  constructor(private RatingsClient: HttpClient) {}
+
+
+  public sendGetRequestRatings():Observable<ProductInterface[]>{
+    return this.RatingsClient.get<ProductInterface[]>(this.REST_API_SERVER);
+  }
+
+  public updateratings(data:any, id:number ):Observable<ProductInterface> {
```

Comment:
this should be updateRatings according to our naming convention

Reply:
Alright. Made the changes.